engines/glk/adrift/detection_tables.h

```diff
@@ -35,10 +35,14 @@ const PlainGameDescriptor ADRIFT_GAME_LIST[] = {
 	{ "30seconds", "30 Second" },
 	{ "3monkeys", "Three Monkeys, One Cage" },
 	{ "achtung", "Achtung Panzer!" },
+	{ "adrbedlam", "Bedlam" },
+	{ "adrbluesky", "Blue Sky" },
+	{ "adrcrimescene", "The Crime Scene" },
```

Comment:
For consistency, these entries should have "adrift" after the gamename:
bedlamadrift , blueskyadrift etc.
You could also change the "asylum" entry to "asylumadrift"

Reply:
You could also do the changes, stage the file and commit --amend
Then force push to your branch and the PR will update automatically

Reply:
I really don't want to make a mess here because I'm still learning git!